docs/zz_generated.kubermaticConfiguration.yaml

```diff
@@ -264,6 +264,9 @@ spec:
       }
     # DockerRepository is the repository containing the Kubermatic dashboard image.
     dockerRepository: quay.io/kubermatic/dashboard
+    # DockerTag is used to overwrite the dashboard Docker image tag and is only for development
```

Comment:
This file needs to be regenerated once the `omitgenyaml` tags are back in place.